test/clojure-mode-sexp-test.el

```diff
@@ -31,30 +31,37 @@
            (wrong))"
         ;; make this use the native beginning of defun since this is used to
         ;; determine whether to use the comment aware version or not.
+        (expect (let ((beginning-of-defun-function nil))
+                  (clojure-top-level-form-p "comment")))))
+  (it "should return true when multiple forms are present"
+    (with-clojure-buffer-point
+        "(+ 1 2) (comment
+           (wrong)
+           (rig|ht)
+           (wrong))"
         (expect (let ((beginning-of-defun-function nil))
                   (clojure-top-level-form-p "comment"))))))
 
 (describe "clojure-beginning-of-defun-function"
   (it "should go to top level form"
     (with-clojure-buffer-point
-      "(comment
+      " (comment
```

Comment:
Why is the space before `comment` needed?

Reply:
I modified it(and a couple of other tests) to ensure `begininng-of-defun` default behaviour is no longer present in the code. With `beginning-of-defun` this test would fail with the additional space.